packager/amp_cache_transform/amp_cache_transform_test.go

```diff
@@ -3,30 +3,52 @@ package amp_cache_transform
 import (
 	"testing"
 
+	"github.com/ampproject/amppackager/transformer"
+	rpb "github.com/ampproject/amppackager/transformer/request"
 	"github.com/stretchr/testify/assert"
 )
 
+func header(header string, version int64) string {
+	return header
+}
+
 func TestShouldSendSXG(t *testing.T) {
```

Comment:
Maybe add some that should break parsing so that it confirms the parsing is WAI.

Reply:
Good idea. Added a bunch of different test cases, but I'm probably missing a bunch more. LMK if you have any in mind.

Reply:
Some additional ones for parseString:

```
`google;v="\`
`google;v="\a`
`google;v="\1`
```

I'd also test the condition `char <= 0x1f || char == 0x7f` but not exactly sure how to do that in a Go test.

Reply:
Done.